src/scripts/quick_reblog.js

```diff
@@ -189,6 +189,7 @@ const processPosts = async function () {
 
     if (alreadyRebloggedList.includes(rootID)) {
       const reblogLink = postElement.querySelector('footer a[href*="/reblog/"]');
+      if (!reblogLink) { return; }
```

Comment:
This is inside a `for` loop rather than a `forEach` callback.

```suggestion
      if (!reblogLink) { continue; }
```

Reply:
It may seem reasonably safe to see one post without a reblog link and assume all unprocessed posts also don't have one, but I'm sure there's a way to break this assumption.

Reply:
...Ah. Well. Now I know how it feels :D And I have to say, it doesn't feel great!